spec/system/jobseekers_can_manage_their_job_alerts_from_the_dashboard_spec.rb

```diff
@@ -3,54 +3,63 @@
 RSpec.describe "Jobseekers can manage their job alerts from the dashboard" do
   let(:jobseeker) { create(:jobseeker) }
 
+  let(:subscriptions_page) { PageObjects::Jobseekers::Subscriptions::Index.new }
+
   context "when logged in" do
-    before do
-      login_as(jobseeker, scope: :jobseeker)
-    end
+    before { login_as(jobseeker, scope: :jobseeker) }
 
     context "when there are job alerts" do
-      before do
-        create(:subscription, email: jobseeker.email, search_criteria: { keyword: "Maths" })
-        visit jobseekers_subscriptions_path
-      end
+      let!(:subscription) { create(:subscription, email: jobseeker.email, search_criteria: { keyword: "Maths" }) }
+
+      before { subscriptions_page.load }
 
-      it "shows their job alerts" do
-        expect(page).to have_content("KeywordMaths")
+      it "shows job alerts" do
+        expect(subscriptions_page.heading).to have_content(I18n.t("jobseekers.subscriptions.index.page_title"))
+        expect(subscriptions_page).to have_cards(count: 1)
+        expect(subscriptions_page.cards[0].header).to have_content("KeywordMaths")
```

Comment:
Why does it have this strange content? I would expect "Keyword: Maths"

Reply:
The `:` isn't part of the content, it's added by css